### PR TITLE
Tiny changes in itn-installer.sh

### PR DESCRIPTION
**Pull request by BoboTiG**

### CODE REVIEW

File: itn-installer.sh [hunk 1]
```diff
@@ -50,7 +50,7 @@ chmod +x /opt/dusk/bin/*
 #mkdir -p /opt/dusk/installer/rusk
 #tar xf /opt/dusk/installer/rusk.tar.gz --directory /opt/dusk/installer/rusk
 #mv /opt/dusk/installer/rusk/rusk /opt/dusk/bin/
-chmod +x /opt/dusk/bin/rusk
+#chmod +x /opt/dusk/bin/rusk
```
Comment (BoboTiG) [inline itn-installer.sh:53]: This is only needed if you uncomment previous lines, else the line 46 is already taking care of setting the proper exec right.

File: itn-installer.sh [hunk 2]
```diff
@@ -78,7 +78,8 @@ systemctl enable rusk
 systemctl daemon-reload
 
 echo "Setup local firewall"
-ufw allow 9000:9005/udp
+ufw allow 8080/tcp
+ufw allow 9000/udp
```
Comment (BoboTiG) [inline itn-installer.sh:82]: Now the local firewall is in sync with the documentation. LMK if that's not a desired change.

File: itn-installer.sh [hunk 3]
```diff
@@ -102,7 +103,4 @@ echo
 echo "To check the logs"
 echo "tail -F /var/log/rusk.{log,err}"
 
-rm -f /opt/dusk/installer/rusk.tar.gz
-rm -f /opt/dusk/installer/installer.tar.gz
-rm -f /opt/dusk/installer/wallet.tar.gz
```
Comment (BoboTiG) [inline itn-installer.sh:107]: The last line of that script will handle those removals automatically.

File: itn-installer.sh [hunk 4]
```diff
@@ -102,7 +103,4 @@ echo
 echo "To check the logs"
 echo "tail -F /var/log/rusk.{log,err}"
```
Comment (BoboTiG) [inline itn-installer.sh:104]: Since 2dc38c7e64c6dee999b56a6d4222473d482595d8:

```suggestion
echo "tail -F /var/log/rusk.log"
```

**Review by herr-seppia — APPROVED** [review]
Thanks for contributing

LGTM

**Comment by BoboTiG** [comment]
Fixed conflicts 👍🏻

**Pull request merged**